### PR TITLE
updated to countlyconsent class

### PR DESCRIPTION
**Pull request by peterBrxwn**

### CODE REVIEW

**Comment by peterBrxwn** [comment]
[[Flutter, doc] Rework our doc to use CountlyConsent for referencing consent](https://trello.com/c/uGCL9Xaf/1418-flutter-doc-rework-our-doc-to-use-countlyconsent-for-referencing-consent)

File: flutter/next.md [hunk 1]
```diff
@@ -1033,7 +1033,7 @@ Countly.disableLocation();</code></pre>
       <span style="font-weight: 400;">when user gets out of temp ID mode</span>
     </li>
     <li>
-      <span style="font-weight: 400;">when 'remote-config' consent is given after it had been removed before (if consents are enabled)</span>
+      <span style="font-weight: 400;">when CountlyConsent.remoteConfig consent is given after it had been removed before (if consents are enabled)</span>
```
Comment (arifBurakDemiray) [inline flutter/next.md:1036]: maybe a code tag to the consent

**Review by arifBurakDemiray — APPROVED** [review]
All seems okey

**Review by turtledreams — CHANGES_REQUESTED** [review]
In User Consent section pls add a sentence or two to tell people that they can use the CountlyConsent inteface to reach all possible consent options and such.
Then ofc you can remove the CountlyConsent prefix in the features list and mention that these are the options you can reach from the interface.

**Comment by peterBrxwn** [comment]
> In User Consent section pls add a sentence or two to tell people that they can use the CountlyConsent inteface to reach all possible consent options and such.
> Then ofc you can remove the CountlyConsent prefix in the features list and mention that these are the options you can reach from the interface.

Done

File: flutter/next.md [hunk 2]
```diff
@@ -1673,27 +1673,28 @@ config.apm.<strong>enableForegroundBackgroundTracking</strong>();</code></pre>
   <a href="/hc/en-us/articles/360037997132" target="_self">can be found here.</a>
 </p>
 <p>
+  You can use CountlyConsent inteface to reach all possible consent options.<br>
```
[the remark this answers is not in the document]
Comment (peterBrxwn) [reply flutter/next.md]: done

File: flutter/next.md (hunk 1 above)
Comment (peterBrxwn) [reply flutter/next.md]: done

**Review by arifBurakDemiray — APPROVED** [review]

**Review by turtledreams — APPROVED** [review]

**Pull request merged**